jsonpb/jsonpb_test.go

```diff
@@ -397,6 +397,42 @@ var marshalingTests = []struct {
 	{"BoolValue", marshaler, &pb.KnownTypes{Bool: &wpb.BoolValue{Value: true}}, `{"bool":true}`},
 	{"StringValue", marshaler, &pb.KnownTypes{Str: &wpb.StringValue{Value: "plush"}}, `{"str":"plush"}`},
 	{"BytesValue", marshaler, &pb.KnownTypes{Bytes: &wpb.BytesValue{Value: []byte("wow")}}, `{"bytes":"d293"}`},
+	{"ListValue nil", marshaler, &pb.KnownTypes{Listval: nil}, `{}`},
```

Comment:
Looking at the repeated tests, I'm not sure if I understand the situation. Repeated fields are clearly supported, so what is the difference between "ListValue" and repeated field?